homeassistant/components/websocket_api/messages.py

```diff
@@ -154,7 +156,7 @@ def cached_state_diff_message(iden: int, event: Event[EventStateChangedData]) ->
         (
             _partial_cached_state_diff_message(event)[:-1],
             b',"id":',
-            str(iden).encode(),
+            iden_bytes,
             b"}",
         )
```

Comment:
I left `construct_result_message` as is since we only construct that once. Its the event messages we can send 10000s+ of times (as long as the UI remains open)